Makefile

```diff
@@ -1,18 +1,22 @@
 CACHITO_COMPOSE_ENGINE ?= docker-compose
+PYTHON_VERSION_VENV ?= python3.8
+PYTHON_VERSION_TOX ?= py38
+PYTHON_TEST_SUITE ?=
 ifeq ($(CACHITO_COMPOSE_ENGINE), docker-compose)
 	DOWN_OPTS=-v
 endif
 
-all: run
+all: venv run-start
 
-clean:
-	$(CACHITO_COMPOSE_ENGINE) down $(DOWN_OPTS)
+clean: run-down
 	rm -rf venv && rm -rf *.egg-info && rm -rf dist && rm -rf *.log* && rm -rf .tox && rm -rf tmp
 
+.PHONY: venv
 venv:
-	virtualenv --python=python3 venv && venv/bin/python setup.py develop && venv/bin/pip install -r requirements.txt -r requirements-web.txt tox
+	virtualenv --python=${PYTHON_VERSION_VENV} venv && venv/bin/python setup.py develop && venv/bin/pip install --upgrade pip && venv/bin/pip install -r requirements.txt -r requirements-web.txt tox
```

Comment:
heads-up: changed in #341 (requirements.txt have to be installed before running `setup.py develop`)

Reply:
Done.